README.md

```diff
@@ -54,10 +54,19 @@ Stream<Row> rowStream = Parsers
   .parse(FileSource.of(new FileInputStream("myFile.xlsx")));
 ```
 
-With CSV files, it is possible to specify multiple parameters like the charset or the escape character:
+With CSV files, it is possible to specify multiple parameters the escape character:
 ```java
 Stream<Row> rowStream = Parsers
-  .csv(CsvParserConfig.builder().charset(StandardCharsets.UTF_8).separator(';').build())
+  .csv(CsvParserConfig.builder().separator(';').build())
+  .parse(FileSource.of(new FileInputStream("myFile.csv")));
+```
+
+The CSV parser will try to detect the encoding charset, but you can specify the fallback charset if none has been found.
+It should always be a charset without BOM, for instance BomCharset.UTF_8_NO_BOM,
+otherwise the length of the BOM will be skipped
```

Comment:
`otherwise the content of the CSV file will be stripped of the length of the BOM.`